NATS.Client.Core.Tests/SendBufferTest.cs

```diff
@@ -143,7 +143,11 @@ void Log(string m)
         Log($"[C] publishing close (8MB)...");
         var pubTask = nats.PublishAsync("close", new byte[1024 * 1024 * 8], cancellationToken: cts.Token).AsTask();
 
+#if NETFRAMEWORK
+        await pubTask;
+#else
         await pubTask.WaitAsync(cts.Token);
+#endif
```

Comment:
I think you can do that instead; you are already passing the cancellation token to the operation, unless you are testing a case I am missing.

```suggestion
        await nats.PublishAsync("close", new byte[1024 * 1024 * 8], cancellationToken: cts.Token);
```